packages/muze-legend/src/legend/simple-legend.js

```diff
@@ -183,7 +186,15 @@ export default class SimpleLegend {
         const effBorder = border * 2;
         const effMargin = margin * 2;
 
-        this.data(this.dataFromScale(this.scale()));
+        // defining scaleParams
+        const scaleParams = {
+            smartLabel: this.labelManager(),
+            measures: this.measurement(),
+            alignment: this.config().position,
+            minTickDistance: this.minTickDistance()
+        };
+
+        this.data(this.dataFromScale(this.scale(), scaleParams));
```

Comment:
Why is scaleParams still being sent?

Reply:
removed the parameters from this.dataFromScale() function.